coldfront/core/user/forms.py

```diff
@@ -28,6 +29,20 @@ class UserSearchForm(forms.Form):
     search_by.widget.attrs.update({'rows': 4})
 
 
+class UserSearchListForm(forms.Form):
+    FIRST_NAME = 'First Name'
+    MIDDLE_NAME = 'Middle Name'
+    LAST_NAME = 'Last Name'
+    USERNAME = 'Cluster Username'
+    EMAIL = 'Email'
+
+    first_name = forms.CharField(label=FIRST_NAME, max_length=100, required=False)
+    middle_name = forms.CharField(label=MIDDLE_NAME, max_length=100, required=False)
+    last_name = forms.CharField(label=LAST_NAME, max_length=100, required=False)
+    username = forms.CharField(label=USERNAME, max_length=100, required=False)
+    email = forms.EmailField(label=EMAIL, max_length=100, required=False)
```

Comment:
Thanks for adding support for other email addresses. Let's add a `help_text` field here with the following text:

"You may use this field to find the user that an email address belongs to, even if it is not a primary address."